packages/components/organisms/f-form/src/mixin/_tests/formValidations.mixin.test.js

```diff
@@ -0,0 +1,306 @@
+
+        describe('isValidPhoneNumber ::', () => {
+            afterEach(() => {
+                jest.clearAllMocks();
+            });
+
+            it('should call `isValidPhoneNumber` from `f-services`', () => {
+                // Arrange
+                const isValidPhoneNumberSpy = jest.spyOn(validations, 'isValidPhoneNumber');
+
+                const wrapper = shallowMount(baseComponent, {
+                    i18n,
+                    localVue
+                });
+
+                // Act
+                wrapper.vm.isValidPhoneNumber();
+
+                // Assert
+                expect(isValidPhoneNumberSpy).toHaveBeenCalledWith(phoneNumberData.value, 'en-GB');
+            });
+        });
+
+        describe('isValidPostcode ::', () => {
+            afterEach(() => {
+                jest.clearAllMocks();
+            });
+
+            it('should call `isValidPostcode` from `f-services`', () => {
+                // Arrange
+                const isValidPostcodeSpy = jest.spyOn(validations, 'isValidPostcode');
+
+                const wrapper = shallowMount(baseComponent, {
+                    i18n,
+                    localVue
+                });
+
+                // Act
+                wrapper.vm.isValidPostcode();
+
+                // Assert
+                expect(isValidPostcodeSpy).toHaveBeenCalledWith(postcodeData.value, 'en-GB');
+            });
+        });
+
+        describe('fieldStatus ::', () => {
+            describe('when field does not have validations', () => {
+                beforeEach(() => {
+                    baseComponent.data = () => ({
+                        formData: {
+                            formFields: [
+                                fieldWithoutValidations
+                            ]
+                        }
+                    });
+                });
+
+                it('should return false', () => {
+                    // Arrange
+                    const wrapper = shallowMount(baseComponent, {
+                        i18n,
+                        localVue
+                    });
+
+                    // Act
+                    const result = wrapper.vm.fieldStatus('lastName');
+
+                    // Assert
+                    expect(result).toEqual(false);
+                });
+            });
+
+            it('should return false when field is not dirty', () => {
+                // Arrange
+                const wrapper = shallowMount(baseComponent, {
+                    i18n,
+                    localVue,
+                    mocks: {
+                        $v: {
+                            formFields: {
+                                fieldWithBothValidations: {
+                                    $dirty: false
+                                }
+                            }
+                        }
+                    }
+                });
+
+                // Act
+                const result = wrapper.vm.fieldStatus('email');
```

Comment:
Is `'email'` important here, or should it be `'fieldWithBothValidations'`? 🤔

Reply:
Oh I see, `fieldWithBothValidations` is of type `email`, so maybe pass in `fieldWithBothValidations.name` here instead?